packages/LablePicture/app/page.tsx

```diff
@@ -0,0 +1,247 @@
+        setComponentStructure: updateComponentStructure,
+        componentIndex: 0,
+        sectionIndex: 0,
+        setTextData,
+        textData: allContents,
+        setProp,
+        mergeProps,
+        setStructureVal,
+        generateNewTextKeys,
+        insertMultipleContentVals,
+      }}
+    >
+      <main>
+        <section className="m-auto container rounded-md leading-0.4 rounded-b-none">
+          <h1 className="text-2xl px-3">
+            <strong>iClasser</strong> <code>Component</code> Playground
+          </h1>
+          <p className="px-3">
+            This is a playground for you to test your new components.<br/> To create a component run <code className='bg-gray-100 px-1 rounded-md'>npx create-iclasser-react-component</code>
+          </p>
+        </section>
+
+        <section className="m-auto container border border-gray-500 bg-slate-100 min-h-screen p-4">
+          <div className="flex justify-center mb-2">
+            <button
+              className="bg-blue-500 text-white px-4 py-2 rounded-md"
+              onClick={() => {
+                setIsEditMode(!isEditMode);
+              }}
+            >
+              {isEditMode ? "Preview" : "Edit"}
+            </button>
+          </div>
+          <div className="flex justify-center items-start h-full">
+            <div className="flex justify-center gap-10 flex-col items-center w-4/6 h-full">
+             
+              <div className="w-full pr-10">
+                <h2 className="text-xl">
+                  {isEditMode ? "Component Editor" : "Component Preview"}
+                </h2>
+                <div className="p-2 border border-gray-500/10 bg-gray-800 rounded-md min-h-96">
+                  {/* Pass isEditMode to LabelComponent */}
+                  <LabelComponent isEditMode={isEditMode} />
+                </div>
+              </div>
+            </div>
+
+            <div className="w-2/6 flex flex-col gap-4">
+              <div>
+                Structure
+                <Editor
+                  height="40vh"
+                  value={JSON.stringify(structureComponent, null, 2)}
+                  className="border border-blue-500 rounded-md p-2 bg-blue-300"
+                  defaultLanguage="json"
+                  options={{ readOnly: true }}
+                  defaultValue={JSON.stringify(structureComponent, null, 2)}
+                />
+              </div>
+
+              <div>
+                Text Data
+                <Editor
+                  height="24vh"
+                  value={JSON.stringify(allContents, null, 2)}
+                  className="border border-blue-500 rounded-md p-2 bg-blue-300"
+                  defaultLanguage="json"
+                  options={{ readOnly: true }}
+                  defaultValue={JSON.stringify(allContents, null, 2)}
+                />
+              </div>
+
+              <div>
+                Coding contents
+                <Editor
```

Comment:
Seems like this is old. get a new copy of the page.tsx and maybe the whole app from another component like MatchingPairs